go/systests/ephemeral_test.go

```diff
@@ -12,7 +12,7 @@ import (
 	context "golang.org/x/net/context"
 )
 
-func TestNewTeamEKNotif(t *testing.T) {
+func TestEphemeralNewTeamEKNotif(t *testing.T) {
```

Comment:
these changes were just so we could run `go test --run TestEphemeral` to verify server side changes more easily